src/reviewer.py

```diff
@@ -23,11 +23,8 @@ def review_pull_request(self):
                 return
 
 
-            # Post all comments in a single batch
-            if review_comments:
-                post_review_comment(review_comments, diff_content)
-            else:
-                print("No valid review comments generated")
+            print(f"Generated {len(review_comments)} potential comments")
+            post_review_comment(review_comments, diff_content)
 
```

Comment:
Print statement before calling post_review_comment.
```suggestion
|
```
N/A